OptimizelySDK/Config/HttpProjectConfigManager.cs

```diff
@@ -122,14 +122,18 @@ protected override ProjectConfig Poll()
         
         public class Builder
         {
+            private const long MAX_MILLISECONDS_LIMIT = 4294967294;
```

Comment:
Cool, we need to make sure that we document these values for options in the README in your other PR

Reply:
> Are these compat suite tests that we can include with this fix to ensure that other SDKs aren't similarly affected?

Sure, we can add FSC test cases.

Reply:
Sure, will add this value in README.md